sql/sql.go

```diff
@@ -0,0 +1,250 @@
+// Package sql provides an way to create sql.DB objects out of an RQLite store.
+//
+// The goal of this package is to provide an interface that looks and feels
```

Comment:
I like this idea in principle. It's probably the interface I should have had the `Store` object implement in the first place. (Though how to do transactions is what you're trying to solve here0.